src/main/webapp/WEB-INF/views/creationForm.jsp

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Developer Form</title>
+  <%@ page import="org.example.constant.UrlPath" %>
+  <style>
+  body {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    margin: 0;
+  }
+
+  form {
+    width: 300px;
+  }
+
+  section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  h2 {
+    text-align: center;
+  }
+
+  label {
+    margin-bottom: 5px;
+  }
+
+  input {
+    width: 100%;
+    padding: 8px;
+    margin-bottom: 10px;
+    box-sizing: border-box;
+  }
+
+  button {
+    width: 100%;
+    padding: 10px;
+    box-sizing: border-box;
+  }
+  </style>
+</head>
+
+<body>
+
+  <form action="<%=UrlPath.CREATE_DEVELOPER.getFullUrl()%>" method="POST">
+      <section>
+        <h2>New developer information</h2>
+          <label for="firstName">
+            <span>First Name: </span>
+          </label>
+          <input type="text" name="First Name" required />
```

Comment:
The `name` attribute should be the same as in `RequestArgument` enum for all `input` elements